support/Tests/GraphQLSupportTests/IntegrationTests.swift

```diff
@@ -17,6 +17,11 @@ class IntegrationTests: XCTestCase {
         XCTAssertEqual(String(describing: query), "{keys(first:10,after:\"cursor\")}")
     }
 
+    func testOptionalArrayQuery() {
+        let query = Generated.buildQuery { $0.mget(keys: ["one", "two"]) }
+        XCTAssertEqual(String(describing: query), "{mget(keys:[\"one\",\"two\"])}")
+    }
```

Comment:
These query tests are mostly testing the arguments, so it isn't really relevant to the change, but if you want to keep it `testArrayArgQuery` might be a more appropriate test name.